File: survival-shooter-unity-demo/Assets/rudder-sdk/Event/RudderEventPayload.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

namespace com.rudderlabs.unity.library.Event
{
    internal class RudderEventPayload
    {
        [JsonProperty(PropertyName = "sent_at")]
        internal string timestamp;
        [JsonProperty(PropertyName = "batch")]
        internal List<RudderEvent> events;

        [JsonProperty(PropertyName = "writeKey")]
        internal string writeKey = "1P6q8fcXrkmekovCdk0a3gFq30X";

        internal RudderEventPayload(List<RudderEvent> events)
        {
            timestamp = DateTime.UtcNow.ToString("u");
            this.events = events;
        }
    }
}

File: old-references/rudder-client-android-kotlin-old/src/main/java/com/rudderlabs/android/library/models/RudderEventPayload.kt
package com.rudderlabs.android.library.models

import com.rudderlabs.android.library.util.Utils
import com.google.gson.annotations.SerializedName

data class RudderEventPayload(
    @SerializedName("sent_at")
    val timestamp: String = Utils.getDate(),
    @SerializedName("batch")
    val events: List<RudderEvent>
)